modules/tinymce/src/themes/silver/main/ts/ui/sidebar/Sidebar.ts

```diff
@@ -101,10 +101,12 @@ const setSidebar = (sidebar: AlloyComponent, panelConfigs: SidebarConfig, showSi
     Replacing.set(slider, [ makeSidebar(panelConfigs) ]);
 
     // Show the default sidebar
-    if (Type.isString(showSidebar)) {
+    if (Type.isString(showSidebar) && Obj.has(panelConfigs, showSidebar)) {
```

Comment:
Doing the `lowerCase` check here would make more sense and would be more foolproof as this could be called from different places.